packages/client/src/v2-events/features/events/registered-fields/Checkbox.tsx

```diff
@@ -40,7 +40,7 @@ function CheckboxInput({
 }
 
 function CheckboxOutput({ value }: { value?: Stringifiable }) {
-  return value === 'true' ? 'Yes' : 'No'
+  return value?.toString() === 'true' ? 'Yes' : 'No'
```

Comment:
this fixes a bug that I noticed, where if the checkbox is toggled it will always display 'No'

Reply:
The function has an outdated signature I think. I should be 

```ts
function CheckboxOutput({ value }: { value?: boolean }) {
```